src/astra_driver.cpp

```diff
@@ -39,11 +39,13 @@
 #include <sys/shm.h>  
 #include <sensor_msgs/image_encodings.h>
 #include <sensor_msgs/distortion_models.h>
+#include <nodelet/nodelet.h>
+#include <ros/ros.h>
```

Comment:
<ros/ros.h> is already included in astra_driver.h, should it be included again here?